diff -Naurw ./arch/arm/boot/dts/stm32mp157c-dk2.dts ../kernel-source/arch/arm/boot/dts/stm32mp157c-dk2.dts
--- ./arch/arm/boot/dts/stm32mp157c-dk2.dts	2021-04-21 10:01:38.409734576 +0200
+++ ../kernel-source/arch/arm/boot/dts/stm32mp157c-dk2.dts	2021-04-21 10:02:17.449810861 +0200
@@ -58,7 +58,13 @@
 	};
 
 	panel_otm8009a: panel-otm8009a@0 {
+#if 0
 		compatible = "orisetech,otm8009a";
+#else
+		compatible = "raspberrypi,rpi-dsicomp";
+		i2c-adapter = <&i2c1>;
+		i2c-addr = <0x45>;
+#endif
 		reg = <0>;
 		reset-gpios = <&gpioe 4 GPIO_ACTIVE_LOW>;
 		power-supply = <&v3v3>;
